src/jit/gentree.h

```diff
@@ -1459,9 +1454,10 @@ struct GenTree
             case GT_INTRINSIC:
             case GT_LEA:
             case GT_STOREIND:
-            case GT_INITBLK:
-            case GT_COPYBLK:
-            case GT_COPYOBJ:
+            case GT_BLK:
+            case GT_OBJ:
```

Comment:
A comment explaining what Op2 represents when we have GT_BLK or GT_OBJ as I would normally think of these two a GT_IND and thus they don't have an Op2.